nsdb-core/src/main/scala/io/radicalbit/nsdb/actors/ShardReaderActor.scala

```diff
@@ -251,6 +270,14 @@ object ShardReaderActor {
 
   case object RefreshShard extends NSDbSerializable
 
+  /**
+    * Stores the results from primary aggregations
+    * @param quantities single quantities (count, sum etc.).
+    * @param uniqueValues unique values that will be used for count distinct calculation.
+    */
+  case class PrimaryAggregationResults(quantities: Map[String, NSDbNumericType] = Map.empty,
+                                       uniqueValues: Set[NSDbType] = Set.empty)
```

Comment:
Do we need default values here? I ask because it looks like we rarely use empty constructor. We could prepare a factory method `def empty()` that returns an empty `PrimaryAggregationResults`.

What do you think about it?

Reply:
We do not use it at all. I guess it can be removed. Thanks for pointing it out.